test-infrastructure/run_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+mkdir -p ~/junit
+python -m pytest $PYTEST_COMMON_OPTIONS
+
+#python -m pytest $PYTEST_COMMON_OPTIONS --splits 2 --group 1 --splitting-algorithm least_duration &
```

Comment:
```suggestion
#  to split work across containers and run tests in parallel like circleci use commands like:
#python -m pytest $PYTEST_COMMON_OPTIONS --splits 2 --group 1 --splitting-algorithm least_duration &
```

Reply:
The tricky thing here is that it the parallel tests don't work, because there's only one database. On circle-ci, it's running 4 complete stacks, so you can get the 4x speedup. Here the database test collide and fail badly.